src/layouts/layout.module.scss

```diff
@@ -0,0 +1,31 @@
+@import "../css/typography.css";
+@import "../css/colors.scss";
+
+body {
+  font-family: 'gilroy', serif;
+}
+
+.navMenuContainer {
+  position: relative;
+  z-index: 10;
```

Comment:
Why this value? Maybe you can define variables.sccs file for this

Reply:
In other PR